### ISSUE TITLE
Building iotedge for Ubuntu 20.04 amd64

### ISSUE THREAD

**Issue by gri6507** [Azure/iotedge#4392]
Following instructions on https://github.com/Azure/iotedge/blob/master/edgelet/doc/devguide.md I am trying to build iotedged DEB package for Ubuntu 20.04

```bash
git clone --recurse-submodules 'https://github.com/Azure/iotedge'
cd iotedge/
PACKAGE_OS='ubuntu20.04' PACKAGE_ARCH='amd64' ./edgelet/build/linux/package.sh 
```

This errors out with `Unrecognized target [ubuntu20.04.amd64]`

Looking at `edgelet/build/linux/package.sh` the problem is clear - there is no 20.04 listed there.

**Comment by gri6507** [comment]
It appears that the 1.1.0 release (just got a notification about it) still does not support Ubuntu 20.04, however current master does. What is the timeline to have a released version which supports 20.04?

**Comment by veyalla** [comment]
The docs are being updated, but with 1.1 release, following [install instructions for Ubuntu 18.04 ](https://docs.microsoft.com/en-us/azure/iot-edge/how-to-install-iot-edge?view=iotedge-2018-06)on a 20.04 system should work without changes.

**Comment by arsing** [comment]
I'm not sure we're planning on building 20.04 packages for 1.1. master will be 1.2.

As Venkat said, you should be able to install 18.04 packages on it anyway.

Since you're building from source, you can patch the script to add the `ubuntu20.04` cases to every existing `ubuntu18.04` case.

**Comment by arsing** [comment]
We've been making 20.04 packages for 1.2, so this is done.

**Comment by Supriyaj29** [comment]
Hello, even  I am having same issue while install Azure IoTedge 1.2 on Ubuntu 20.04. 

As per the documentation, I added prod.list for 18.04 using following command
`curl https://packages.microsoft.com/config/ubuntu/18.04/multiarch/prod.list > ./microsoft-prod.list`
I tried replacing 18.04 with 20.04, it return 404 Resource Not Found error.

And If I continue with 18.04 list, apt-get update fails with below error:

W: GPG error: https://packages.microsoft.com/ubuntu/18.04/multiarch/prod bionic InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 
E: The repository 'https://packages.microsoft.com/ubuntu/18.04/multiarch/prod bionic InRelease' is not signed.

Have I missed anything? Would really appreciate some help on this. Thanks.
